crates/indexer/Cargo.toml

```diff
@@ -7,9 +7,9 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-gw-types = { git = "https://github.com/nervosnetwork/godwoken.git", rev = "08e2a451a74653cf8b9899aaa693d3aa91623337" }
-gw-common = { git = "https://github.com/nervosnetwork/godwoken.git", rev = "08e2a451a74653cf8b9899aaa693d3aa91623337" }
-gw-jsonrpc-types = { git = "https://github.com/nervosnetwork/godwoken.git", rev = "08e2a451a74653cf8b9899aaa693d3aa91623337" }
+gw-types = { git = "https://github.com/classicalliu/godwoken.git", rev = "c6573fa0a8a5f080086de82c47f20ddd13e78202" }
```

Comment:
it this pr depends on un-merge updates on godwoken? can we mount to the officail godwoken repo?

Reply:
Updated